tests/test-umockdev-record.vala

```diff
@@ -71,9 +71,9 @@ t_testbed_all_empty ()
 
     spawn ("umockdev-record" + " --all", out sout, out serr, out exit);
 
-    assert_cmpstr (serr, Op.EQ, "");
-    assert_cmpstr (sout, Op.EQ, "");
-    assert_cmpint (exit, Op.EQ, 0);
+    Assertions.assert_cmpstr (serr, Op.EQ, "");
```

Comment:
I tried to drop umockdev's assertions.vapi, and with that I get tons of
```
../../source/tests/test-umockdev-run.vala:436.5-436.18: error: The name `assert_cmpuint' does not exist in the context of `t_gphoto_download'
    assert_cmpuint ((uint) st.st_size, Op.GT, 5000);
```

There still only seems to be [GLib.assert()](https://valadoc.org/glib-2.0/GLib.assert.html), and a few non-comparison ones like [GLib.assert_error()](https://valadoc.org/glib-2.0/GLib.assert_error.html). We could probably use the [standard Check vapi](https://valadoc.org/check/Check.assert_str_eq.html), but last time I tried them they were not as nice as glib's.

Reply:
Hmm, it could be because I have a custom compiled glib inside jhuild.

I am seeing:
```
../tests/test-umockdev-record.vala:74.5-74.17: error: `assert_cmpstr' is an ambiguous reference between `GLib.assert_cmpstr' and `Assertions.assert_cmpstr'
    assert_cmpstr (serr, Op.EQ, "");
    ^^^^^^^^^^^^^
```

Maybe the collision is because for newer vala/glib it is exported in the `.vapi`?

Reply:
So yeah, just checked and from vala 0.51.1 (there is no 0.51.0) onwards these functions are supported.

So, probably sane to just add a version check instead.

Reply:
Nice! You just fixed this in #119, so resolving.